test/test_api.py

```diff
@@ -156,14 +174,38 @@ def test_get_profile(test_client: FlaskClient) -> None:
     response = test_client.get('/profiles/a1b2c3d4')
 
     assert response.status_code == 200
+    assert response.headers.get('Cache-Control') == 'max-age=300, public, stale-if-error=86400,' \
+                                                    'stale-while-revalidate=300'
     assert response.headers.get('Content-Type') == 'application/vnd.elife.profile+json;version=1'
+    assert response.headers.get('Vary') == 'Accept'
 
     data = json.loads(response.data.decode('UTF-8'))
 
     assert validate_json(data, schema_name='profile.v1') is True
     assert data['id'] == 'a1b2c3d4'
 
 
+def test_get_profile_revalidation(test_client: FlaskClient) -> None:
+    profile = Profile('a1b2c3d4', Name('Foo Bar'), '0000-0002-1825-0097')
+
+    db.session.add(profile)
+    db.session.commit()
```

Comment:
would extract the 3 lines into something like `_add_profile('a1b2c3d4')`

Reply:
I have added some `@cache_control_headers` decorators and pushed them to this branch. Let me know if they fit what we need.

They will probably need renaming and moving into a decorators module of some sort, but its a start.